server/database/schema.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE user(
   id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
   email VARCHAR(100) UNIQUE NOT NULL,
-  password VARCHAR(64) NOT NULL
+  hashed_password VARCHAR(150) 
```

Comment:
Pourquoi diable avez vous retirez le not null sur le mot de passe ?!?!

Si il n'y pas de mot de passe c'est problématique non ? 🤔

Reply:
done